core/src/components/textarea/textarea.tsx

```diff
@@ -310,29 +313,31 @@ export class Textarea implements ComponentInterface {
           [mode]: true,
         }}
       >
-        <textarea
-          class="native-textarea"
-          ref={el => this.nativeInput = el}
-          autoCapitalize={this.autocapitalize}
-          autoFocus={this.autofocus}
-          disabled={this.disabled}
-          maxLength={this.maxlength}
-          minLength={this.minlength}
-          name={this.name}
-          placeholder={this.placeholder || ''}
-          readOnly={this.readonly}
-          required={this.required}
-          spellCheck={this.spellcheck}
-          cols={this.cols}
-          rows={this.rows}
-          wrap={this.wrap}
-          onInput={this.onInput}
-          onBlur={this.onBlur}
-          onFocus={this.onFocus}
-          onKeyDown={this.onKeyDown}
-        >
-          {value}
-        </textarea>
+        <div class="textarea-wrapper">
```

Comment:
Can we add a ref here instead of doing a `querySelector` every time we run autogrow?

```suggestion
        <div class="textarea-wrapper"
          ref={el => this.textareaWrapper = el}
        >
```

Then instead of doing `const inputContainer = this.el.querySelector<HTMLElement>('.textarea-wrapper');` you can just do `const inputContainer = this.textareaWrapper`